com.unity.ml-agents/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Major Changes
 #### com.unity.ml-agents (C#)
 #### ml-agents / ml-agents-envs / gym-unity (Python)
+- TensorFlow trainers have been removed, please use the Torch trainers instead. (#4707)
```

Comment:
This will have to move to move to the 'Unreleased' section after release 11 (unless we are aiming to get this in for the release)